src/mapblock.h

```diff
@@ -154,6 +154,11 @@ class MapBlock /*: public NodeContainer*/
 		raiseModified(MOD_STATE_WRITE_NEEDED, MOD_REASON_REALLOCATE);
 	}
 
+	MapNode* getData()
```

Comment:
Can't do that; there are so many non-const interfaces everywhere.

I tried it.